components/create/CreateCollection.vue

```diff
@@ -276,37 +281,4 @@ onMounted(() => {
 })
 </script>
 
-<style lang="scss" scoped>
-@import '@/styles/abstracts/variables';
-
-.o-field:not(:last-child) {
-  margin-bottom: 2rem;
-}
-
-.column {
-  max-width: 36rem;
-  padding: 4rem;
-
-  @include desktop() {
-    @include ktheme() {
-      background-color: theme('background-color');
-      box-shadow: theme('primary-shadow');
-    }
-  }
-
-  @include touch() {
-    padding: 0 1rem;
-    box-shadow: none !important;
-  }
-}
-
-@include desktop() {
-  .columns {
-    padding: 5.25rem 0;
-
-    @include ktheme() {
-      background-color: theme('k-primaryLight');
-    }
-  }
-}
-</style>
+<style lang="scss" scoped src="@/styles/pages/create.scss"></style>
```

Comment:
what is the advantage of this style vs the regular
```vue
<style>
@import ...
</style>

Reply:
reduce loc?

Reply:
i guess

Reply:
I feel like the template/script on `CreateNft/CreateCollection` is too long, just in case we want to split that into separate file. Especially after adding `Listing NFT` function on rmrk

```html
<template src="./template.html"></template>
<script src="./script.js"></script>
```